internal/services/appconfiguration/app_configuration_feature_resource_test.go

```diff
@@ -480,6 +502,27 @@ resource "azurerm_app_configuration_feature" "test" {
 `, t.template(data), data.RandomInteger)
 }
 
+func (t AppConfigurationFeatureResource) basicTargetingFilter(data acceptance.TestData) string {
```

Comment:
Thanks for adding the test, looks like a required argument is missing here though:

```
------- Stdout: -------
=== RUN   TestAccAppConfigurationFeature_basicAddTargetingFilter
=== PAUSE TestAccAppConfigurationFeature_basicAddTargetingFilter
=== CONT  TestAccAppConfigurationFeature_basicAddTargetingFilter
    testcase.go:113: Step 3/4 error: Error running pre-apply refresh: exit status 1
        Error: Missing required argument
          on terraform_plugin_test.tf line 59, in resource "azurerm_app_configuration_feature" "test":
          59:   targeting_filter {
        The argument "default_rollout_percentage" is required, but no definition was
        found.
```